libalpm/scripts/mkinitcpio-install

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
I think we really should add `-e` here so that errors abort the script.

Reply:
Solved both in 59bdecbf138936f1eb61c9616d7270d92a5deac5